### PR TITLE
use builtin google proto files

### PR DESCRIPTION
**Pull request by hicqu**
Signed-off-by: qupeng <qupeng@pingcap.com>

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -29,7 +29,8 @@ default-features = false
 features = ["secure", "use-bindgen"]
 
 [build-dependencies]
-protobuf-build = { version = "0.12", default-features = false }
+# protobuf-build = { version = "0.12", default-features = false }
+protobuf-build = { git = "https://github.com/hicqu/protobuf-build", branch = "include-google", default-features = false }
```
Comment (tisonkun) [inline Cargo.toml:33]: Is this a pr ready to review? I don't know why we depend on a personal fork. It looks brittle.

File: include/rustproto.proto [hunk 2]
```diff
@@ -1,47 +0,0 @@
-syntax = "proto2";
```
Comment (solotzg) [inline include/rustproto.proto:1]: `./scripts/generate_cpp.sh` failed after this file deleted.

**Comment by disksing** [comment]
Do we need to proceed with this pull request? cc @hicqu

**Comment by disksing** [comment]
closing for inactivity.

**Pull request closed without merging**